### ISSUE TITLE
TalkBack: FAB.Group breaks select with touch

### ISSUE THREAD

**Issue by personalizedrefrigerator** [callstack/react-native-paper#4064]
### Current behaviour

When using TalkBack on Android, exploring the screen by touch [(docs)](https://support.google.com/accessibility/android/answer/6006598?hl=en) doesn't work in apps that use a `FAB.Group`. Selecting items by clicking on them also doesn't work.

### Expected behaviour

Touch exploration/selection should work.

### How to reproduce?

https://github.com/personalizedrefrigerator/bug-report/tree/react-native-paper-broken-fab

### Preview

https://github.com/callstack/react-native-paper/assets/46334387/e51d4a30-fb2c-43e6-a00b-fed3eae20c09

### What have you tried so far?
1. Using an empty `<Portal>` element, rather than one containing a `FAB.Group` (works)
2. Using a `<Portal>` element that displays a dialog (works)
3. Using an absolutely-positioned `FAB`, rather than a `FAB.Group` (works)

### Your Environment

| software                      | version
| --------------------- | -------
| ios                                | untested
| android                        | 10
| react-native                | 0.72.4
| react-native-paper    | 5.10.4
| node                            | 18.13.0
| npm or yarn                | 9.6.7 (npm)
| expo sdk                     | N/A

**Comment by personalizedrefrigerator** [comment]
The issue seems to be related to the use of `ponter-events="box-none"` on the FAB.Group container:
https://github.com/callstack/react-native-paper/blob/813cdd47a90ee3bb693c6ec143af73092cd3edae/src/components/FAB/FABGroup.tsx#L344-L346

It seems to be possible to work around this issue by changing the size of the container when the FAB group is collapsed, as [is done here](https://github.com/laurent22/joplin/pull/10123/files). (Note: The linked example adjusts the container size using hardcoded values, which could be problematic).
